CapybaraLearning/capybara.rb

```diff
@@ -0,0 +1,19 @@
+require 'rubygems'
+require 'capybara'
+require 'capybara/poltergeist'
+
+describe 'Searching with only delivery tests' do
```

Comment:
not sure, why we need this?

Reply:
oh, it was test file). I`ll remove this one.